src/ffi/playermove.rs

```diff
@@ -110,46 +181,43 @@ pub struct playermove_s {
     pub PM_PointContents: unsafe extern "C" fn(p: *mut f32, truecontents: *mut c_int) -> c_int,
     pub PM_TruePointContents: unsafe extern "C" fn(p: *mut f32) -> c_int,
     pub PM_HullPointContents:
-        unsafe extern "C" fn(hull: *mut c_void, num: c_int, p: *mut f32) -> c_int,
+        unsafe extern "C" fn(hull: *mut hull_s, num: c_int, p: *mut f32) -> c_int,
     pub PM_PlayerTrace: unsafe extern "C" fn(
         start: *const f32,
         end: *const f32,
         traceFlags: TraceFlags,
         ignore_pe: c_int,
     ) -> pmtrace_s,
     pub PM_TraceLine: unsafe extern "C" fn(
-        start: *const f32,
-        end: *const f32,
-        flags: TraceLineFlag,
+        start: *mut f32,
+        end: *mut f32,
+        flags: c_int,
```

Comment:
I think this should remain as it was?

Reply:
I did not change all of these. It is the bindgen that changed it. But yea, I understand why it is more desireable for it to be const.

Reply:
Yeah, I believe I changed those by hand to what they are on current main